handlers/whatsapp/whatsapp.go

```diff
@@ -776,8 +776,12 @@ func (h *handler) getTemplate(msg courier.Msg) (*MsgTemplating, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "invalid templating definition")
 	}
+	// check country
+	if templating.Country != "" {
+		templating.Language = fmt.Sprintf("%s_%s", templating.Language, templating.Country)
+	}
 
-	// map our language from iso639-3 to the WA country / iso638-2 pair
+	// map our language from iso639-3_iso3166-2 to the WA country / iso638-2 pair
 	language, found := languageMap[templating.Language]
```

Comment:
Here we will then try to look up in our language map using `eng_US` instead of just `eng` no?

Reply:
There is a check below this code, converting the `language` to the `language_country` format, if template has a country.

Reply:
Note that the `language` will be as `eng_US` if template has a country, otherwise it will be as `eng`.

Reply:
And both `eng` and `eng_US` are in the `languageMap`.